app/client/service_runner.py

```diff
@@ -1,7 +1,8 @@
-import requests
 from subprocess import DEVNULL
 import time
 
+import requests
+
```

Comment:
There's a new lint rule `wrong-import-order` that enforces the pep8 recommendation that third party lib imports be separated from stdlib imports. Intellij can fix this automatically with "Optimize Imports" and that's how I fixed all these.